setWallpaper.py

```diff
@@ -19,7 +22,16 @@ def change_windows_background(filename):
 
     SPI_SETDESKWALLPAPER = 0x14 #which command (20)
     SPIF_UPDATEINIFILE   = 0x2 #forces instant update
-    print(ctypes.windll.user32.SystemParametersInfoW(SPI_SETDESKWALLPAPER, 
-                                                     0, 
-                                                     filename, 
+    print(ctypes.windll.user32.SystemParametersInfoW(SPI_SETDESKWALLPAPER,
+                                                     0,
+                                                     filename,
```

Comment:
i dont mind linting changes and its ok for now but genereally on big repos it goes in another commit :) its fine for us btw :D